### PR TITLE
Feat/MYNW-154 Added Explore page

### PR DESCRIPTION
**Pull request by followdarko**
Figma design https://www.figma.com/file/qVgRY71P3XjttByfO6prmG/Explore-A?node-id=1%3A175

Added:

* Explore page /explore
* NavBar link to explore
* Light sidebar banner on the wallet page

### CODE REVIEW

**Comment by roshaans** [comment]
Is it possible to use SVG images or higher res pictures where possible? Some icons are very pixelated like the Rainbow bridge icon and Burrow Icon.

File: packages/frontend/src/components/wallet/SidebarLight.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    useEffect(() => {
+        if (availableAccounts?.length > 0) {
+            const numNonImplicitAccounts = availableAccounts.filter((a) => a.length < 64).length;
+            setActiveComponent(numNonImplicitAccounts === 0 ? 'CreateCustomName' : 'ExploreApps');
+        }
+    }, [availableAccounts]);
+
+    return (
+        <StyledContainer>
+            <StyledBanner>
```
Comment (roshaans) [inline packages/frontend/src/components/wallet/SidebarLight.js:55]: UI Nit: Seems to be a bit of inconsistency between the figma design and the layout. 

image

**Review by roshaans — COMMENTED** [review]
Everything besides the nit picky details I mentioned look good :) 

Before shipping, I think it would be a great idea to get hi res pictures of the logos like AwesomeNear.com has.

File: packages/frontend/src/components/explore/ExploreContainer.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+import { selectAvailableBalance } from '../../redux/slices/account';
+import { formatNearAmount } from '../common/balance/helpers';
+import Container from '../common/styled/Container.css';
+import Banner from './components/Banner';
+import ExploreSection from './components/ExploreSection';
+import TrandingProjects from './components/TrandingProjects';
+import { trandingProjects, exchangeSection, startEarningSection, collectNFTsSection, playToEarnSection } from './content';
+
+const StyledContainer = styled(Container)`
```
[the remark this answers is not in the document]
Comment (followdarko) [reply packages/frontend/src/components/explore/ExploreContainer.js:14]: Fixed

File: packages/frontend/src/components/wallet/SidebarLight.js (hunk 1 above)
Comment (followdarko) [reply packages/frontend/src/components/wallet/SidebarLight.js:56]: Tunned banner resolution.
Fixed banner margins.
Fixed some icons on Explore page

**Review by roshaans — APPROVED** [review]
Terrific job :) Changes look great. 

LGTM

**Pull request merged**